src/applications/check-in/utils/navigation/index.js

```diff
@@ -5,6 +5,8 @@ const isWithInDays = (days, pageLastUpdated) => {
   return daysAgo <= days;
 };
 
+const travelAllowList = ['0001'];
```

Comment:
Should we maybe bubble this up to [appConstants](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/check-in/utils/appConstants/index.js)? I'm open. I think it would also be good to document the location and how it works in the main or day-of README.

Reply:
Good idea!